### PR TITLE
Update 03-generators.mdx

### PR DESCRIPTION
**Pull request by m1212e**
I made [prismabox](https://github.com/m1212e/prismabox), a generator to generate typebox schemes from the prisma scheme. Found it very useful when used with elysia and since there is no typebox tool listed yet, I think it would help people in the same situation

### CODE REVIEW

File: content/200-orm/100-prisma-schema/10-overview/03-generators.mdx [hunk 1]
```diff
@@ -96,3 +96,5 @@ The following is a list of community created generators. If you want to create y
 - [`prisma-markdown`](https://github.com/samchon/prisma-markdown): Generates markdown document composed with ERD diagrams and their descriptions. Supports pagination of ERD diagrams through `@namespace` comment tag.
 - [`prisma-models-graph`](https://github.com/dangchinh25/prisma-models-graph): Generates a bi-directional models graph for schema without strict relationship defined in the schema, works via a custom schema annotation.
 - [`prisma-generator-fake-data`](https://github.com/luisrudge/prisma-generator-fake-data): Generates realistic-looking fake data for your Prisma models that can be used in unit/integration tests, demos, and more.
+- [`prismabox`](https://github.com/m1212e/prismabox): Generates versatile [typebox](https://github.com/sinclairzx81/typebox) schemes from your Prisma models.
```
Comment (jharrell) [inline content/200-orm/100-prisma-schema/10-overview/03-generators.mdx]:
```suggestion
- [`prismabox`](https://github.com/m1212e/prismabox): Generates versatile [typebox](https://github.com/sinclairzx81/typebox) schema from your Prisma models.
```

**Comment by jharrell** [comment]
Hi there! Thank you for your contribution (both to our docs and to our community) 🙂 

I see that your repo was forked from another project. Is that project still maintained? Generally speaking, we'd appreciate it if custom generators matched out suggested naming scheme of `prisma-generator-<custom name>`. 

To summarize:
- it's unclear to me if https://github.com/adeyahya/prisma-typebox-generator is still maintained. If it's not, would it be possible for you to take over the maintenance of that project? If it is maintained, could you pass your updates to the upstream project and we can list that?
- we prefer generators to have the naming scheme of `prisma-generator-<name>`. In this case it would be `prisma-generator-typebox`. (I understand that is pretty confusing with `prisma-typebox-generator` still existing, so if you're unable to get in contact with the maintainers of that project, nevermind this)

**Review by jharrell — CHANGES_REQUESTED** [review]
blocked pending discussion around related project.

**Comment by m1212e** [comment]
I see. I opened a few issues asking for features/bugfixes but never got a response, so I think it's not maintained anymore.

I really like "prismabox" :D Fits so good. But if thats your policy I see why a rename would make sense.

**Review by jharrell — APPROVED** [review]

**Pull request merged**